apps/expert/rel/overlays/bin/start_expert.bat

```diff
@@ -1,5 +1,4 @@
 @echo off
 
-cd /d "%~dp0"
-.\plain.bat eval "System.no_halt(true); Application.ensure_all_started(:xp_expert)" --stdio
+"%~dp0plain.bat" eval "System.no_halt(true); Application.ensure_all_started(:xp_expert)" --stdio
```

Comment:
I forgot to pass the args in the windows file instead of hardcoding `--stdio`
```suggestion
"%~dp0plain.bat" eval "System.no_halt(true); Application.ensure_all_started(:xp_expert)" %*
```